RestApi/Sepes.RestApi.Tests/Mocks/PodMock.cs

```diff
@@ -45,5 +45,10 @@ public Task<string> GetPods(int studyID)
         {
             return Task.FromResult(JsonSerializer.Serialize(new Pod(42, "name", studyID, false, new List<Rule>(), new List<Rule>(), new List<User>(), new List<DataSet>(), new List<DataSet>())));
         }
+
+        public Task Set(Pod newPod, Pod based)
+        {
+            throw new NotImplementedException();
```

Comment:
A function that do not return anything can often be mocked by an empty function. An empty async function in this case.